### DISCUSSION TITLE
simulation doesn't start

### DISCUSSION THREAD

**Discussion by silviliril** [GeoChemFoam/GeoChemFoam#2] [General]
I am trying to do the tutorial of rising bubble. But, the simulation doesn't start. What can be wrong? The case file is attached. I get following error. (I use GeoChemFoam-4.5)
`Create time

Create mesh for time = 0

SIMPLE: no convergence criteria found. Calculations will run for 0.04 steps.

PIMPLE: Operating solver in PISO mode

Reading g
Reading field pd

Reading field alpha1

Reading field U

Reading/calculating face flux field phi

Reading transportProperties

Selecting incompressible transport model Newtonian
Selecting incompressible transport model Newtonian
Reading thermoPhysicalPropertiesProperties

Create species mixture

Reading compressive flux coefficient

Calculating field g.h

Selecting turbulence model type laminar
Hello from continuityErrs
time step continuity errors : sum local = 0, global = 0, cumulative = 0
GAMG:  Solving for pcorr, Initial residual = 0, Final residual = 0, No Iterations 0
Hello from continuityErrs
time step continuity errors : sum local = 0, global = 0, cumulative = 0
Courant Number mean: 0 max: 0 velocity magnitude: 0

Starting time loop

Courant Number mean: 0 max: 0 velocity magnitude: 0
deltaT = 1.19998e-06
Time = 1.19998e-06

PIMPLE: iteration 1
Floating point exception (core dumped)
`
[risingBubble2.tar.gz](https://github.com/GeoChemFoam/GeoChemFoam/files/8604292/risingBubble2.tar.gz)

**Comment by GeoChemFoam** [comment]
Hi @silviliril. I'm taking a look now, are you using the precompiled docker or have you installed GeoChemFoam 4.5 natively on your system?

**Reply by silviliril** [reply]
Thank you for the reply. 

I downloaded the GeoChemFoam-4.5 from github and compiled it on my linux after installing OpenFOAM-4.x and foamExtend-4.0.

**Reply by julienmaes** [reply]
Hi Liril,

The way this tutorial is set-up, the bubble is already initialized. We know it is a problem and we will modify it for future versions. In the meantime, if you want to initialize the bubble differently using setFields, make sure you set-up the species concentration as well (Field T), otherwise, it will be 0 everywhere and that will make the solver crash.
